README.md

```diff
@@ -141,6 +142,20 @@ exclude:
 >rake preview:all
 >```
 
+## Docker (docker-compose)
```

Comment:
To add some context to this feature, can you add the following information in a summary before the instructions:

1. What are the use cases for using this feature?
1. Do users need anything else besides docker installed?
1. Will this work from a fresh copy of the repo or do people need to perform some pre-requisite steps?

Reply:
It's quite simple. Only a docker installation is needed. Should work in every repo/fork. I can add an additional comment.

Reply:
@jcalcaben I added an description. Please review it. I am not a native speaker.